### DISCUSSION TITLE
Guide for deployment on Google Cloud Platform Kubernetes Engine

### DISCUSSION THREAD

**Discussion by jmhbnz** [kimai/kimai#2627] [On-Premise: Installation & Updates] [answered]
Hi Everyone - I recently stumbled across Kimai while looking at options for a small business time sheeting application and wrote a brief guide for running the application on Google Kubernetes Engine: https://github.com/jmhbnz/kimai-gcp-deployment.

Thought I would post back here in case anyone else wants to deploy Kimai on Kubernetes. 

Many thanks to the maintainer and contributors for putting this project together, what a fantastic application.

**Comment by kevinpapst** [comment, marked as answer]
Awesome stuff and short feedback from my side:
- If you add some `kimai` tags to your repository it can be easily found.
- Would be cool if you could prepare and send a [Pull request for the installation docs](https://github.com/kimai/www.kimai.org/blob/master/_documentation/installation.md) - I'd say a new section around Ansible with a link to your repo?!

Edit: Sorry, can't say anything to the actual implementation, because that is something I have never used, but it looks well prepared 👍

**Reply by jmhbnz** [reply]
Sounds good, have added tags and raised PR: https://github.com/kimai/www.kimai.org/pull/103 

If you see any issues in future with people needing support for Kubernetes deployments feel free to tag me in them, happy to assist!
